docs/reference/commandline/create.md

```diff
@@ -69,6 +69,7 @@ Creates a new container.
       --ulimit=[]                   Ulimit options
       --uts=""                      UTS namespace to use
       -v, --volume=[]               Bind mount a volume
+      --volume-driver=""            Optional volume driver for the container
```

Comment:
Is `Optional` needed here? These are all options on this list.

Container's volume driver.